src/io/flutter/run/MainFile.java

```diff
@@ -120,16 +121,22 @@ public boolean hasFlutterImports() {
 
   @Nullable
   private static VirtualFile findAppDir(@Nullable VirtualFile file, @NotNull Project project) {
+    if (WorkspaceCache.getInstance(project).isBazel()) {
+      Workspace workspace = WorkspaceCache.getInstance(project).get();
+      if (workspace != null) {
```

Comment:
you should assert `workspace != null` instead of using an if.
`isBazel()` is equivalent to `workspace != null`.